.github/workflows/codecoverage.yml

```diff
@@ -1,7 +1,8 @@
 name: Code Coverage (Apollo tests)
 on:
+  workflow_dispatch:
   schedule:
-    - cron: "0 3 * * 0" # Every Sunday at 3AM UTC
+    - cron: "0 20 * * 6" # Every Saturday at 8PM UTC
```

Comment:
why change the job schedule?

Reply:
Hi @teoparvanov,
Because all of the booked jobs are scheduled on Saturday so I decided to schedule this job for later on Saturday.
Also, all of the scheduled jobs will be completed without conflict on Saturday.